test/logger-errors.js

```diff
@@ -0,0 +1,267 @@
+'use strict'
+
+const {test} = require('tap')
+const nock = require('nock')
+const {Logger} = require('../lib/logger.js')
+const configs = require('../lib/configs.js')
+const {apiKey, createOptions} = require('./common/index.js')
+
+nock.disableNetConnect()
+
+test('timestamp validity checks', async (t) => {
+  const startTime = Date.now()
+  const logger = new Logger(apiKey, createOptions())
```

Comment:
Random style nit: any reason you're using `const logger = new Logger()` in some tests and `const log = new Logger()` in some others?

Reply:
yeah, because `log.log` looked dumb AF :). I probably just didn't fix the first ones I had done since using a level like `log.warn` does make sense.

Reply:
I can change them all to `logger`

Reply:
whoops, the file is also called `logger.js`, so that creates a problem.

Reply:
I'll think I'll just leave these if it's all the same.  It's a lot to change.